templates/es2015/server.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Module dependencies.
+ */
+
+import app from './app';
+import {default as debugModule} from 'debug';
```

Comment:
Here's an example, if you open both these links you'll see the generated code is identical.

* https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&code=import%20foo%20from%20'foo'
* https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&code=import%20%7Bdefault%20as%20foo%7D%20from%20'foo'